firefox-ios/Client/Frontend/Browser/Menu/MainMenuViewModel.swift

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import Shared
+
+class MainMenuViewModel {
```

Comment:
Just to confirm, this view model is temporary until replaced with the redux middleware, right?

Reply:
yas.